### PR TITLE
Add.unit.test.suite

### PR DESCRIPTION
**Pull request by herman-rogers**
Just adding more unit tests to the configs and rewriting them to make it slightly more robust and dynamic.

### CODE REVIEW

File: poller/poller.go [hunk 1]
```diff
@@ -1,69 +1,83 @@
...
-    if err != nil {
-        logger.Warning(err.Error())
-        return;
-    }
-    messages := response.Messages
-    if(len(messages) > 0) {
-        InboundMessages(session, messages, process)
-        return;
-    }
+	response, err := session.ReceiveMessage(params)
```
Comment (chvck) [inline poller/poller.go]: Are these meant to be tabs?

**Comment by chvck** [comment]
Might be worth squashing some of these commits as they do similar things?

File: poller/poller.go (hunk 1 above)
Comment (herman-rogers) [reply poller/poller.go]: I just let the google go fmt thing take over. I can switch it to spaces.

Comment (herman-rogers) [reply poller/poller.go]: how can you tell if they're tabs or spaces?

Comment (chvck) [reply poller/poller.go]: The spaces are green and if you highlight them theyre tabs

**Pull request merged**